src/OmniSharp.Abstractions/Models/v1/FilesChanged/FilesChangedRequest.cs

```diff
@@ -3,6 +3,9 @@
 
 namespace OmniSharp.Models.FilesChanged
 {
-    [OmniSharpEndpoint(OmniSharpEndpoints.FilesChanged, typeof(IEnumerable<Request>), typeof(FilesChangedResponse))]
-    public class FilesChangedRequest : IRequest { }
+    [OmniSharpEndpoint(OmniSharpEndpoints.FilesChanged, typeof(IEnumerable<FilesChangedRequest>), typeof(FilesChangedResponse))]
+    public class FilesChangedRequest : Request
+    {
+        public FileChangeType ChangeType { get; set; }
```

Comment:
This will serialize as an int (I think, I didn't see any where we configure `Newtonsoft.Json`)